x-pack/plugins/security_solution/public/management/pages/blocklist/services/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './blocklists_api_client';
```

Comment:
Maybe I'm wrong but I think we had some issues with the wildcard export in the past, if it's not needed (I don't think so), can we export just what we want?